designer/server/src/main/scala/pl/touk/nussknacker/ui/api/description/scenarioActivity/Dtos.scala

```diff
@@ -586,6 +586,7 @@ object Dtos {
         scenarioVersionId: Option[Long],
         comment: ScenarioActivityComment,
         dateFinished: Option[Instant],
+        status: Option[String],
```

Comment:
Changed to sealed trait with statuses. Serialized to human-readable string when converting to dto for HTTP API.

Reply:
It is of different type in the domain - ProcessActionState, not related to the state of periodic deployment. I decided to leave it as primitive type, because ProcessActions are not yet replaced by ScenarioActivities in the Nu code.